Tests/TuistCacheTests/ContentHashing/DeploymentTargetContentHasherTests.swift

```diff
@@ -27,50 +27,51 @@ final class DeploymentTargetContentHasherTests: TuistUnitTestCase {
 
     func test_hash_whenIosIphoneV1_callsContentHasherWithExpectedStrings() throws {
         // When
-        let deploymentTarget = DeploymentTarget.iOS("v1", .iphone, supportsMacDesignedForIOS: false)
+        //, .iphone, supportsMacDesignedForIOS: false
```

Comment:
What is this?

Reply:
ah, nice catch, this was a note to myself about the original implementation of the `DeploymentTargets` initializer that was used here so i could make sure the new implementation matched.  I'll get this cleaned up